sql/analyzer/resolve_columns.go

```diff
@@ -538,14 +629,23 @@ func resolveColumns(ctx *sql.Context, a *Analyzer, n sql.Node, scope *Scope, sel
 			return nil, transform.SameTree, err
 		}
 
-		return transform.OneNodeExprsWithNode(n, func(n sql.Node, e sql.Expression) (sql.Expression, transform.TreeIdentity, error) {
+		newN, same, err := transform.OneNodeExprsWithNode(n, func(n sql.Node, e sql.Expression) (sql.Expression, transform.TreeIdentity, error) {
 			uc, ok := e.(column)
 			if !ok || e.Resolved() {
 				return e, transform.SameTree, nil
 			}
 
 			return resolveColumnExpression(a, n, uc, columns)
 		})
+
+		// Ignore errors when trying to resolve JSONTable, will be handled later
+		if err != nil {
```

Comment:
kind of concerning, maybe should be opaque but we'll see what zach thinks

Reply:
I'm curious what kind of errors occur here, and when they get eventually resolved

Reply:
When we do something similar for subqueries, they eventually fail because we have a rule that treats errors as final.

Think we need tests for json tables with unresolvable column references. You don't have any failure test cases right now